### ISSUE TITLE
Licensing Module reporting incorrect status

### ISSUE THREAD

**Issue by jungeunyoon** [Rise-Vision/licensing-module#10]
Here's what we know:

- For the sake of our tests from Twitter Component we ping Licensing Module through Twitter Module via LM every second or so for Licensing status.
- Licensing Module is consistently returning that some displays aren't authorized - yesterday it was 1/10 displays and today it was 3/10. 
  - 4U3Q2KXEAD9G
  - 89YMNGW3J8SE
  - XRM5W486T7WT
- After toggling Rise Player Professional on Apps and rebooting the media player for all affected displays the Licensing status started to report that the affected displays were authorized.
**- We want to say that all of these displays' Rise Player Professional licenses were indeed active (took a cursory glance at the logs) but you might want to rule that one out first because maybe it was just doing what it was supposed to be doing?**

Thanks!

**Comment by tejohnso** [comment]
@jungeunyoon It sounds like it's doing what it's supposed to be doing. 

Can you confirm whether there's a discrepancy between what licensing is reporting vs `https://store-dot-rvaserver2.appspot.com/v1/widget/auth?id=${displayId}&pc=c4b368be86245bf9501baaa6e0b00df9719869fd&startTrial=false`

Also, can you confirm that you're not going to be pinging in production?

**Comment by jungeunyoon** [comment]
@tejohnso Correct, each component will be attempting a maximum of 5 times per start should we not receive a Licensing message. Do you see a problem with that?

After a discussion we will be closing this issue as there are too many unknowns - it's not currently reproducible and all the displays are currently working so there isn't a display to point you to. We'll continue to monitor as we launch Twitter and keep you guys updated if anything comes up. Thanks!
